### DISCUSSION TITLE
RENOVATE_ONBOARDING_COMMIT_MESSAGE Question

### DISCUSSION THREAD

**Discussion by marais-vzc** [renovatebot/renovate#28485] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab 37.302.0

### Please tell us more about your question or problem

We deployed renovate setting `RENOVATE_ONBOARDING_COMMIT_MESSAGE` with a value of `chore: add renovate.json`

The problem is see is that renovate would create the commit message like `Chore: add renovate.json`. This is unexpected as the commits are being rejected because we have a regex on GitLab that ensures specific patterns for commit messages, and a capital C for chore is not allowed and somewhat not aligned with conventional commits either. 

![image](https://github.com/renovatebot/renovate/assets/161749428/0904215a-2862-4a06-94e3-aec227baf5f6)

![image](https://github.com/renovatebot/renovate/assets/161749428/92f716be-f842-482d-858f-4406d09d430b)

Is this a bug or is there a way to change this behaviour?

### Logs (if relevant)

<details><summary>Logs</summary>

```

To https://git.host/path/to/repo/amazon-linux/amazon-linux-2/amazon_linux_2_arm64.git
!	refs/heads/renovate/configure:refs/heads/renovate/configure	[remote rejected] (pre-receive hook declined)
Done
Pushing to https://git.host/path/to/repo/amazon-linux/amazon-linux-2/amazon_linux_2_arm64.git
POST git-receive-pack (633 bytes)
remote: GitLab: Commit message does not follow the pattern '^((build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test)(\(\w+\))?(!)?(: (.*\s*)*))|(Merge (.*\s*)*)|(Initial commit$)'
error: failed to push some refs to 'https://git.host/path/to/repo/amazon-linux/amazon-linux-2/amazon_linux_2_arm64.git'

Error: To https://**redacted**@3.24.0/node_modules/simple-git/src/lib/plugins/error-detection.plugin.ts:42:29)
    at PluginStore.exec (/usr/local/renovate/node_modules/.pnpm/simple-git@3.24.0/node_modules/simple-git/src/lib/plugins/plugin-store.ts:54:29)
    at /usr/local/renovate/node_modules/.pnpm/simple-git@3.24.0/node_modules/simple-git/src/lib/runners/git-executor-chain.ts:124:42
    at new Promise (<anonymous>)
    at GitExecutorChain.handleTaskData (/usr/local/renovate/node_modules/.pnpm/simple-git@3.24.0/node_modules/simple-git/src/lib/runners/git-executor-chain.ts:121:14)
    at GitExecutorChain.<anonymous> (/usr/local/renovate/node_modules/.pnpm/simple-git@3.24.0/node_modules/simple-git/src/lib/runners/git-executor-chain.ts:97:40)
    at Generator.next (<anonymous>)
    at fulfilled (/usr/local/renovate/node_modules/.pnpm/simple-git@3.24.0/node_modules/simple-git/dist/cjs/index.js:52:24)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)

```

</details>

**Comment by rarkins** [comment]
Confirmed bug here: https://github.com/renovatebot/renovate/issues/22346

Strangely this sounds like the opposite: https://github.com/renovatebot/renovate/issues/25662
